slick.grid.js

```diff
@@ -3120,7 +3128,16 @@ if (typeof Slick === "undefined") {
       addlCssClasses += (formatterResult && formatterResult.addClasses ? (addlCssClasses ? ' ' : '') + formatterResult.addClasses : '');
       var toolTip = formatterResult && formatterResult.toolTip ? "title='" + formatterResult.toolTip + "'" : '';
 
-      stringArray.push("<div class='" + cellCss + (addlCssClasses ? ' ' + addlCssClasses : '') + "' " + toolTip + ">");
+      var customAttrStr = '';
+      if(m.hasOwnProperty('cellAttrs') && m.cellAttrs instanceof Object) {
+        for (var key in m.cellAttrs) {
```

Comment:
I don't think the `for (key of items)` is supported in old browser like IE8 which SlickGrid still support. It's probably better to change that into the regular `for (var i = 0; i < m.cellAttrs.length; i++)`

Reply:
`for (var key in m.cellAttrs)` is supported since IE6. Check out:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in#Browser_compatibility

and 

https://www.w3schools.com/jsref/jsref_forin.asp

Reply:
ahh ok my mistake, it's good then :)